dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/utils/RuntimeUtils.kt

```diff
@@ -1,19 +1,39 @@
 package com.datadog.android.log.internal.utils
 
 import com.datadog.android.BuildConfig
+import com.datadog.android.Datadog
 import com.datadog.android.log.Logger
+import com.datadog.android.log.internal.logger.ConditionalLogHandler
+import com.datadog.android.log.internal.logger.LogcatLogHandler
+import com.datadog.android.log.internal.logger.NoOpLogHandler
 
 internal const val SDK_LOG_PREFIX = "DD_LOG"
+internal const val DEV_LOG_PREFIX = "Datadog"
+
 /**
- * Global SDK Logger. This logger is meant for debugging purposes. Should not post logs
+ * Global SDK Logger. This logger is meant for internal debugging purposes. Should not post logs
  * to Datadog endpoint and should be conditioned by the BuildConfig flag.
  */
-internal val sdkLogger: Logger = buildLogger()
+internal val sdkLogger: Logger = buildSdkLogger()
+
+internal fun buildSdkLogger(): Logger {
+    val handler = if (BuildConfig.LOGCAT_ENABLED) {
+        LogcatLogHandler(SDK_LOG_PREFIX)
+    } else {
+        NoOpLogHandler
+    }
+    return Logger(handler)
+}
+
+/**
+ * Global Dev Logger. This logger is meant for user's debugging purposes. Should not post logs
+ * to Datadog endpoint and should be conditioned by the Datadog Verbosity level.
+ */
+internal val devLogger: Logger = buildDevLogger()
```

Comment:
Shouldn't this be public in order to be accessed by clients ? Maybe we should also write a simple integration test for it ?

Reply:
No it shouldn't be public. This is a  way for us to provide information about the lib to the client. If they need a dev logger that doesn't send data do the servers, they can create it with their own specs (service name, ...)

Reply:
Not sure how we'd do integration tests as it's mostly reporting issues on a lower level :/

Reply:
Yeah this made sense only if that devLogger was meant to be public otherwise it doesn't ;)